src/plots/cartesian/axes.js

```diff
@@ -1604,7 +1604,7 @@ axes.doTicks = function(gd, axid, skipTitle) {
     var axletter = axid.charAt(0),
         counterLetter = axes.counterLetter(axid),
         vals = axes.calcTicks(ax),
-        datafn = function(d) { return d.text + d.x + ax.mirror; },
+        datafn = function(d) { return [d.text, d.x, ax.mirror].join('\u0000'); },
```

Comment:
We used \u0000 as a workaround but putting it on the end of our ticktext values, but not causing any visible change. No solid reason we couldn't use something else here.